sdk/ml/azure-ai-ml/tests/dataset/unittests/test_data_operations.py

```diff
@@ -170,13 +175,17 @@ def test_create_or_update_missing_path(self, mock_data_operations: DataOperation
         Expect to raise ValidationException for missing path
         """
         name = "random_name"
-        data = Data(name=name, version="1", description="this is an mltable dataset", type=AssetTypes.MLTABLE)
+        data1 = Data(name=name, version="1", description="this is an mltable dataset", type=AssetTypes.MLTABLE)
```

Comment:
Is there a reason for this variable renaming?

Reply:
I had a previous commit where I'd added a "data2" variable below, but when I deleted it I forgot to revert this one back to "data". Since it made no difference in function, I just left it.